standalone/README.md

```diff
@@ -35,41 +35,53 @@ If this image is out-of-date, you can edit it [here](https://docs.google.com/dra
 
 ## Setting Up `simple-server`
 
-Below are instructions to setting up simple-server on a set of servers. These instructions assume that you have already
-provisioned servers and have their static IP addresses available. If you don't have servers provisioned yet, you will
-need to do so first.
+These instructions assume that you have already provisioned servers and have their static IP addresses available.
+If you don't have servers provisioned yet, you will need to do so first. See [provisioning testing servers.](#provisioning-testing-servers)
 
-These instructions are to be followed in the `standalone` directory of this repository.
+These instructions are to be followed in the [standalone](/standalone) directory.
+
+### Install ansible
+```bash
+brew install ansible
+```
+### Configure the ansible setup:
 
 - `cd ansible/`
-- Add the IP addresses of your servers to the `hosts/icmr/playground` Ansible inventory file.
+- Create an hosts inventory file. The setup uses `hosts/icmr/playground` as an example. You can use it as a template to setup your own.
+- Add the IP addresses of your servers to the inventory file.
 - Set up your domain and SSL certificate.
     - Add SSL certificates for your domain to `roles/load_balancing/vars/ssl-vault.yml`. This is an encrypted Ansible
       vault file. See [Editing vault files](#editing-vault-files) for instructions on how to edit it.
     - Add the SSL certificate domain names to `haproxy_cert_names` in `group_vars/load_balancing.yml`
     - Configure your DNS records to point your domain/subdomain to the load balancer's IP address. You may do this by
       creating/editing an ALIAS or CNAME record.
-- Set the following in the `hosts/icmr/playground` Ansible inventory file
+- Set the following in the inventory file
     - Set `domain_name` to your domain name (eg. `playground.simple.org`)
```

Comment:
Let's change these domain examples to use `example.com` instead, as it's a reserved domain for documentation/testing.